user/pages/02.development/03.creating-a-skill/docs.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Development Setup
+taxonomy:
+    category:
+        - docs
+---
+
+# Setting up the folders

```

Comment:
For a better organization let's use:
- Header 1 for the main topic: `# Creating a Skill`
- Header 2 for the "step topics": `## Setting up the folders`, `## Creating your first Skill` ...
- Header 3 for the subtopics: `### Dialog`, `### Vocabulary` ...
and so on